### ISSUE TITLE
wsgi-dashboard: Document icon dependency on Awesome Fonts

### ISSUE THREAD

**Issue by rayalan** [buildbot/buildbot#4939]
The latest version of buildbot is built with Awesome Fonts 4.1.0. However, many of the icons displayed at the url listed by the wsgi-dashboard documentation (http://fontawesome.io/icons/) require a newer version. 

For people just trying out the feature, it'd be nice for the documentation to mention this detail and point them to the about feature, which conveniently lists the bundled version of Awesome Fonts.

**Comment by tardyp** [comment]
The version of font-awesome is documented in #about url. We can upgrade it at any moment, so we don't want to maintain to place where it is described

**Comment by rayalan** [comment]
Right...so in the sample code (http://docs.buildbot.net/current/manual/customization.html?highlight=dashboard#writing-dashboards-with-flask-or-bottle), I think this could be as simple as changing: the current sample: ```
# available icon list can be found at http://fontawesome.io/icons/
'icon': 'area-chart'
``` 
to something like: 
```  
# The available icon list can be found at http://fontawesome.io/icons/. 
# Check your buildbot about  page for the applicable version of Font Awesome as the main page 
# lists the icons for the most recent version of Font Awesome, which may not match the current
# buildbot version.
'icon': 'area-chart'
```

**Comment by p12tic** [comment]
@rayalan: Could you create a pull request with this improvement to the documentation?
